terraform/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "gcp_project_id" {
```

Comment:
Add the copyright notice to all the files. ([example](https://github.com/GoogleCloudPlatform/anthos-samples/blob/main/anthos-bm-gcp-terraform/variables.tf#L1-L15)). Make sure to use `2022` as the year